cmake/kokkos_enable_devices.cmake

```diff
@@ -19,10 +19,19 @@ KOKKOS_CFG_DEPENDS(DEVICES NONE)
 KOKKOS_DEPRECATED_LIST(DEVICES ENABLE)
 
 
-KOKKOS_DEVICE_OPTION(PTHREAD       OFF HOST "Whether to build C++ thread backend")
-IF (KOKKOS_ENABLE_PTHREAD)
+KOKKOS_DEVICE_OPTION(THREADS OFF HOST "Whether to build C++ threads backend")
+KOKKOS_OPTION(ENABLE_PTHREAD OFF BOOL "DEPRECATED use Kokkos_ENABLE_THREADS instead!")
```

Comment:
I'm not quite sure if this should still appear as an option in the `ccmake` GUI. Maybe, use `mark_as_advanced`?

Reply:
Good point about showing up as an option.

Reply:
I just removed it.